schema/encoding/jsonschema.go

```diff
@@ -84,7 +84,7 @@ func SchemaToJSONSchema(s *schema.Schema) string {
 			required = append(required, fmt.Sprintf(`"%s"`, key))
 		}
 		if field.ReadOnly {
-			result += fmt.Sprintf("%q: %t, ", "readonly", field.ReadOnly)
+			result += fmt.Sprintf("%q: %s, ", "readonly", `{ "type": "boolean" }`)
```

Comment:
What does JSONSchema do with fields it doesn't recognize? ignore them or fail? If it ignores them I'll just change the case to readOnly and leave it at that.